Bender.yml

```diff
@@ -16,8 +16,9 @@ dependencies:
   hyperbus:           { git: https://github.com/pulp-platform/hyperbus.git,             version: 0.0.4                                } 
   car_l2:             { git: git@iis-git.ee.ethz.ch:carfield/carfield_l2_mem.git,       rev: 0143fb296e3614adf8894ff7335cc3f9b92783bd } # branch: main
   safety_island:      { git: git@iis-git.ee.ethz.ch:carfield/safety-island.git,         rev: cc541a6cf5995eea29dd255db68fbe7d2cd10af6 } # branch: michaero/carfield
-  pulp_cluster:       { git: https://github.com/pulp-platform/pulp_cluster.git,         rev: 8356888056441d6cf5b94e348a0817264b9b1b8c } # branch: yt/rapidrecovery
   opentitan:          { git: https://github.com/pulp-platform/opentitan.git,            rev: 2c2830c055198f75fcd193f0e6772f70c2e2b036 } # branch: carfield_soc
+  pulp_cluster:       { git: https://github.com/pulp-platform/pulp_cluster.git,         rev: 4d1558ac660ad445f62ffe44415ca6efc4240cf6 } # branch: yt/rapidrecovery
+  opentitan:          { git: https://github.com/pulp-platform/opentitan.git,            rev: 4946c1ab9dd0f3191a9dea91edd58aa3647a0e80 } # branch: carfield_soc
```

Comment:
you sure you rebased properly? the opentitan is not tracked in the lock!

Reply:
and should not appear if you rebased onto main

Reply:
wtf

Reply:
fixing this